packages/now-next/src/dev-server.ts

```diff
@@ -10,25 +10,15 @@ process.on('unhandledRejection', err => {
   process.exit(1);
 });
 
-async function main(cwd: string) {
-  const next = require(resolveFrom(cwd, 'next'));
-  const app = next({ dev: true, dir: cwd });
+process.once('message', async ({ dir, runtimeEnv }) => {
+  // eslint-disable-next-line @typescript-eslint/no-var-requires
+  const next = require(resolveFrom(dir, 'next'));
+  const app = next({ dev: true, dir });
   const handler = app.getRequestHandler();
 
-  const openPort = await getPort({
-    port: [5000, 4000],
-  });
-
+  const [openPort] = await Promise.all([getPort(), app.prepare()]);
```

Comment:
Why did you remove the `port: [5000, 4000]` option?

Reply:
Because in reality, we don't care what port the dev-server is listening on. It's an ephemeral port.

Reply:
Ok I'm going to defer to the Next.js team on this one just to be sure

Reply:
This LGTM. We don't care what port the dev server is on!